soc/riscv/riscv-privilege/hpmicro/Kconfig.series

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 HPMicro
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_HPMICRO
+	bool "HPMicro HPM6700_6400 Series"
+	select RISCV
+	select SOC_FAMILY_RISCV_PRIVILEGE
+	select HAS_HPMICRO_HAL
```

Comment:
A SoC doesn't have a HAL, it's drivers that instead select if they need it.

Reply:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/98945035/5fffa49d-e00e-40d8-9b49-7b9a719feffe)

The soc files will be dependent on the hal, and it looks like other chips handle it that way too.